packages/keycloak-user-management/src/components/UserList/index.tsx

```diff
@@ -187,12 +187,13 @@ const UserList = (props: UserListTypes): JSX.Element => {
                 total: isSearchActive ? keycloakUsers.length : usersCount,
               }}
               onChange={(pagination, __, sorter) => {
+                setIsLoading(true);
```

Comment:
@machariamuguku do we need a loader during sort here?

Reply:
@p-netm probably not. @ashfahan what do you think?

Reply:
if we are fetching data from the server i.e load on demand data we need it and if we aren't then we don't need to show any loader

Reply:
638edd0d70d2561b8f9a53bae6b8206d8f9b68b8